internal/providers/pluginfw/resources/cluster/tests/data_cluster_test.go

```diff
@@ -0,0 +1,20 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/databricks/terraform-provider-databricks/internal/acceptance"
+)
+
+func TestAccDataSourceCluster(t *testing.T) {
+	acceptance.WorkspaceLevel(t, acceptance.Step{
```

Comment:
Can we add test cases for the other pathways as well? E.g. cluster by name, multiple clusters with the same name, no such cluster with the given name/ID?